Modify GPT guide

Mani-Sadhasivam opened the pull request:
This commit modifies the GPT guide into two individual guides for SD card and eMMC.

Signed-off-by: Manivannan Sadhasivam <manivannan.sadhasivam@linaro.org>

ndechesne commented:
please give me some time to review this patch, before merging it. this is an important page for Dragonboard. I will make comment if needed, or merge.

ndechesne commented on ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md:
```diff
@@ -0,0 +1,84 @@
+---
+title: Create a bootable SD card
+permalink: /documentation/ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md.html
+redirect_from: /documentation/ConsumerEdition/DragonBoard-410c/Guides/GPT.md/
```
does that mean that the 'old' link will still work? if so, good! if not, then can you please make sure we have a link on the old page? i am sure i have shared this link many times..

Mani-Sadhasivam replied:
I guess the old link will redirect to new page. But I do need @shovanuk to confirm this. Also, one more thing, I have the old guide redirect to both the guides. So, don't know how this will work.

I mean, **redirect from: .GPT.md** is present for both guides. We should remove this from one of the guides. So, eventually one guide will become new guide which doesn't have any previous redirect.

ndechesne commented:
for now, I would keep the old page and add a text that says that content was moved and provide the 2 new pages.

Mani-Sadhasivam commented:
Sure, I'll do that. @shovanuk Can you please address the redirect issue for the 2 new guides?

sdrobertw commented:
@Mani-Sadhasivam Please outline what is to be done with this PR.

redirects can be implemented in the data section at the top of each markdown file, similar to what is done in this README.md

https://github.com/96boards/documentation/blob/master/ConsumerEdition/DragonBoard-410c/README.md

Use this as an example, you should be apply to implement the redirect without @shovanuk 's help.

I would like to merge this one @Mani-Sadhasivam and @ndechesne approve the request.

Thanks!

Mani-Sadhasivam commented:
@ndechesne @shovanuk @sdrobertw 

So, this is what I have done. Just kept the old page and added description that the content has been moved!. On the new guides, I have removed the redirects since we are not deleting the old file.

ndechesne commented:
ok, looks good to me.
